pkgs/tools/networking/ndn-tools/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib
 , stdenv
-, boost
+, boost ? ndn-cxx.boost
```

Comment:
Isn't boost always defined? So this would never occur?

Reply:
Isn't this the way you suggested with fallthrough? I'm not familiar with that option, so I'm not really sure if it does use the same package automatically

Reply:
We need to guarantee that the same version of boost is used both for the NDN library (ndn-cxx) and the applications (ndn-tools, nfd, ...). Is this the most appropriate way?

Reply:
I used the standard boost as suggested